gui/src/components/home/Home.tsx

```diff
@@ -44,7 +44,7 @@ export function Home() {
         </div>
       )}
       {config?.debug && trackers.length > 0 && (
-        <div className="flex px-2 pt-5 justify-center  overflow-x-auto">
+        <div className="px-2 pt-5 justify-center overflow-y-scroll overflow-x-auto">
```

Comment:
```suggestion
        <div className="px-2 pt-5 overflow-y-scroll overflow-x-auto">
```
Seeing the child element, it's just another div that is also a flex box.